src/views/Downloads.vue

```diff
@@ -139,7 +154,15 @@
     watch: {
       $route: 'updateData'
     },
+    computed: {
+      sponsor: function () {
+        return Sponsors[Math.floor(Math.random() * Math.floor(Sponsors.length))];
```

Comment:
I still don't understand why you `Math.floor(Sponsors.length)` instead of just `Math.floor(Math.random() * Sponsors.length)`